src/java.base/share/classes/jdk/internal/javac/PreviewFeature.java

```diff
@@ -74,8 +74,6 @@ public enum Feature {
         SCOPED_VALUES,
         @JEP(number=480, title="Structured Concurrency", status="Third Preview")
         STRUCTURED_CONCURRENCY,
-        @JEP(number=466, title="ClassFile API", status="Second Preview")
-        CLASSFILE_API,
```

Comment:
Just remembered that we have to keep this until JDK 24 is the minimum boot JDK; with this removed bootcycle build should fail.

Reply:
https://github.com/openjdk/jdk/pull/22150